apps/test/unit/templates/sectionsRefresh/SingleSectionSetUp.jsx

```diff
@@ -21,4 +21,37 @@ describe('SingleSectionSetUp', () => {
       .simulate('change', {target: {value: 'Section 1'}});
     expect(updateSectionSpy).to.have.been.calledOnce;
   });
+
+  it('renders MultiSelectGroup with expected props', () => {
+    const wrapper = shallow(
+      <SingleSectionSetUp
+        sectionNum={1}
+        section={{}}
+        updateSection={() => {}}
+      />
+    );
+
+    const multiSelectGroup = wrapper.find('MultiSelectGroup');
+    expect(multiSelectGroup.prop('name')).to.eq('grades');
+    expect(multiSelectGroup.prop('required')).to.eq(true);
+    expect(multiSelectGroup.prop('options').length).to.eq(14); // K + 12 + Other
+    expect(multiSelectGroup.prop('values')).to.eql([]);
+  });
+
+  it('calls updateSection when grade selection is updated', () => {
+    const updateSectionSpy = sinon.spy();
+    const wrapper = mount(
```

Comment:
You need `mount` here because of clicking the checkbox right?

Reply:
i think so, but i'll double check before merging

Reply:
confirmed, it needs mount to find the checkbox element inside the child component

![image](https://user-images.githubusercontent.com/1382374/221691205-1ca6da60-53f4-4501-8803-a7ae1e4ccd0d.png)